apps/examples/testcase/le_tc/network/Kconfig

```diff
@@ -6,7 +6,6 @@
 menuconfig EXAMPLES_TESTCASE_NETWORK
 	bool "Network TestCase Example"
 	default n
-	select NET_LWIP_ERROR if NET_LWIP
```

Comment:
We need to enable NET_LWIP_ERROR, please refer to master branch for the latest changes.

Reply:
@dr-venkman 
Enable NET_LWIP_ERROR , please verify.